internal/osimage/aws/awsupload.go

```diff
@@ -588,7 +588,7 @@ func toPtr[T any](v T) *T {
 
 const (
 	waitInterval    = 15 * time.Second
-	maxWait         = 15 * time.Minute
```

Comment:
Is this here on purpose?

Reply:
Yeah, I agreed to increase the timeout here with Malte since the AWS image upload usually takes longer than the specified 15 minutes currently.